src/app/ngrx/reducers/models/downloads-table.model.ts

```diff
@@ -0,0 +1,12 @@
+import { TimeseriesDatumModel } from 'ngx-prx-styleguide';
+
+export interface DownloadsTableModel {
```

Comment:
This one is actually a view model, not a state model. I don't love having it with the reducer models... It was a last minute change to resolve a TypeScript warning where one of the imports in a component wasn't able to see it in the selector file.